agent/command/results_gotest.go

```diff
@@ -29,11 +29,6 @@ type goTestResults struct {
 	OptionalOutput   string `mapstructure:"optional_output" plugin:"expand"`
 	outputIsOptional bool
 
-	// Optional, when set to true, causes this command to be skipped over without an error when
-	// no test output contains no tests
-	OptionalTests    string `mapstructure:"optional_tests" plugin:"expand"`
```

Comment:
I assume no because it's not documented, but will removing this break YAMLs that have it set?

Reply:
No because [it hasn't been deployed yet](https://github.com/evergreen-ci/evergreen/commit/d39c13a670b8916682642c4abca814368d427bce)

Reply:
Ima merge this but let me know if you have other questions about it!